### ISSUE TITLE
`function Phoenix.LiveView.TagEngine.init/1 is undefined (module Phoenix.LiveView.TagEngine is not available)`

### ISSUE THREAD

**Issue by michaelterryio** [phenixdigital/phoenix_storybook#279]
Just added a storybook to my project a couple days ago and it mostly works, but clicking on a core component in the sidebar gives:

![2023-04-02_19-02-38](https://user-images.githubusercontent.com/57774951/229395352-c9d0ed4c-20cc-436e-b97c-7a2c965a355a.png)

**Comment by cblavier** [comment]
Ok, I'll try to reproduce it and let you know.

**Comment by michaelterryio** [comment]
It looks like TagEngine was released in phoenix_live_view in 0.18.17 and the current release for phoenix_storyview (0.5.2) requres >0.18.17 for phoenix_live_view.

My mix.lock has phoenix_live_view at 0.18.16, so that explains the bug. I fixed it by incrementing the phoenix_live_view version in my mix and mix.update-ing.

Personally, I find it suprirsing that mix lets you install a package with a transitive dependency whose version requirement is higher than one you already have installed.

Unless anyyone has any more light to shed, I guess we're free to close ✅
